curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/check-for-mixed-grouping-of-characters.english.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 5c3dda8b4d8df89bea71600f
+title: Check For Mixed Grouping of Characters
+challengeType: 1
+---
+
+## Description
+<section id='description'>
+Sometimes we want to check for groups of characters using a Regular Expression, to achieve this task we use the round brackets <code>()</code>.
+If you want to find either <code>"Penguin"</code> or <code>"Pumpkin"</code> in a string, you could use the following Regular Expression: <code>/P(engu|umpk)in/g</code>
+Then you can test the code using the <code>test()</code> method to find whether the requires character(s)/string(s) are in the test string or not.
```

Comment:
```suggestion
Then check whether the desired string groups are in the test string by using the <code>test()</code> method.
```

Reply:
Do we want to write string groups or just strings?